packages/paste-website/src/pages/components/multiselect-combobox/index.mdx

```diff
@@ -0,0 +1,360 @@
+import {Text} from '@twilio-paste/text';
+import {MultiselectCombobox} from '@twilio-paste/combobox';
+import {Button} from '@twilio-paste/button';
+import {Table, THead, TBody, Td, Th, Tr} from '@twilio-paste/table';
+import {MediaObject, MediaFigure, MediaBody} from '@twilio-paste/media-object';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {LinkExternalIcon} from '@twilio-paste/icons/esm/LinkExternalIcon';
+import {ProductStudioIcon} from '@twilio-paste/icons/esm/ProductStudioIcon';
+import {ProductAutopilotIcon} from '@twilio-paste/icons/esm/ProductAutopilotIcon';
+import {ProductInsightsIcon} from '@twilio-paste/icons/esm/ProductInsightsIcon';
+import {SearchIcon} from '@twilio-paste/icons/esm/SearchIcon';
+import {CloseIcon} from '@twilio-paste/icons/esm/CloseIcon';
+import {AttachIcon} from '@twilio-paste/icons/esm/AttachIcon';
+import {UnorderedList, ListItem} from '@twilio-paste/list';
+import {Callout, CalloutHeading, CalloutText} from '@twilio-paste/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import filter from 'lodash/filter';
+import {SidebarCategoryRoutes} from '../../../constants';
+import Changelog from '@twilio-paste/combobox/CHANGELOG.md';
+import {
+  basicExample,
+  insertBeforeAfterExample,
+  optionGroupExample,
+  optionGroupLabelTemplateExample,
+  optionTemplateExample,
+  disabledExample,
+  disabledOptionsExample,
+  errorExample,
+  emptyStateExample,
+  maxHeightExample,
+} from '../../../component-examples/MultiselectComboboxExamples';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/combobox"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(frontmatter: {slug: {eq: "/components/multiselect-combobox/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "MultiselectCombobox"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/combobox"
+  storybookUrl="/?path=/story/components-combobox-multiselectcombobox--multiselect-combobox-basic"
+  data={props.data}
+  name="Multiselect Combobox"
+  description="A multiselect combobox is a styled dropdown form element that allows users to select multiple values from a list."
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview scope={{MultiselectCombobox}} noInline language="jsx" showOverflow>
```

Comment:
Issue(blocking): Missing box and text in scope so this example breaks when it shows the empty state.